### PR TITLE
[IN-1974] Packer up the BaseOS Catalina 10.15.7

### PR DESCRIPTION
**Pull request by bitrise-tom**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

File: packer/run_packer.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+if test z$1 = z ; then
+  echo "Please pass a packer json file to build"
+  exit 2
+fi
+
+export PACKER_LOG=1
+export ANKA_DEFAULT_PASSWD=vagrant
+export ANKA_DEFAULT_USER=vagrant
+
+sudo -E packer build -debug -color=true -timestamp-ui $1
```
Comment (lmesz-bitrise) [inline packer/run_packer.sh]: As far as I remember if packer is called with `-debug` it expects user interaction. I think for automation it should be removed.

**Review by lmesz-bitrise — CHANGES_REQUESTED** [review]

File: packer/run_packer.sh (hunk 1 above)
Comment (lmesz-bitrise) [reply packer/run_packer.sh]: oh :)

**Review by lmesz-bitrise — DISMISSED** [review]

File: packer/baseOS.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+     "installer_app": "/Applications/Install macOS Catalina.app/",
+     "disk_size": "{{user `disk_size`}}",
+     "ram_size": "{{user `ram_size`}}",
+     "cpu_count": "{{user `cpu_count`}}",
+     "vm_name": "{{ user `vm_name` }}",
+     "source_vm_name":  "{{user `source_vm`}}"
+    }
+  ],
+  "provisioners": [
+    {
```
Comment (fehersanyi-bitrise) [inline packer/baseOS.json:14]: do we really need this precheck?

**Review by fehersanyi-bitrise — APPROVED** [review]

File: packer/baseOS.json (hunk 2 above)
Comment (bitrise-tom) [reply packer/baseOS.json:14]: it was really useful for debugging just to make sure we are the vagrant user - it's an extra 10 seconds

**Review by fehersanyi-bitrise — APPROVED** [review]

**Review by lmesz-bitrise — APPROVED** [review]

**Pull request merged**